src/main/java/seedu/address/logic/parser/UntagCommandParser.java

```diff
@@ -30,6 +29,10 @@ public UntagCommand parse(String args) throws ParseException {
         ArgumentMultimap argMultimap = ArgumentTokenizer.tokenize(args,
                 PREFIX_TAG);
 
+        if (argMultimap.getAllValues(PREFIX_TAG).size() > 1) {
+            throw new ParseException("Error: More than one 't/' detected. Please use only one 't/' for untagging.");
```

Comment:
Should also add an error check for if the input for untag is some non alphanumeric character eg. `untag *`